### PR TITLE
XIVY-13508: Add placeholder to monaco-editor

### PR DESCRIPTION
**Pull request by ivy-edp**
I have implemented the option to add a placeholder to the Monaco editor. Currently, I have structured it so that the placeholder text can be passed into the CodeEditor as a parameter. At the moment, the placeholder "Press CTRL + SPACE for auto-completion" is simply present in all CodeEditorCells. However, it might make sense to set this placeholder for all singleLineCodeEditors or even for all Monaco editors. What do you think?
beispiel_placeholder

### CODE REVIEW

**Comment by ivy-edp** [comment]
Unfortunately, there isn't a built-in placeholder option in the Monaco editor, so I had to invent this functionality myself using a reference and styling. Here are the two sources I found on this topic and used as references:
https://github.com/microsoft/monaco-editor/issues/568
https://stackblitz.com/edit/react-ts-6egvre?file=App.tsx

**Review by ivy-lli — APPROVED** [review]
Ahh crazy that this is not built in, but I think a fine solution 👍 
I think the placeholder should be always the same on each editor. So I think there is no need for a property.

**Comment by ivy-edp** [comment]
Perfect, than I will remove all the placeholder-properties and write the Placeholder "Press CTRL + SPACE for auto-completion" directly into the placeholder-div inside the code editor. So now not only all SingleLineEditors have this place holder but all the other monaco-editors too.

File: packages/editor/src/components/widgets/code-editor/CodeEditor.css [hunk 1]
```diff
@@ -25,5 +25,10 @@
   font-family: Consolas, 'Courier New', monospace;
   pointer-events: none;
   user-select: none;
-  z-index: 10;
+  z-index: 2;
+}
+
+.code-editor .with-lineNumbers {
+  left: 35px;
+  font-size: 12px;
```
Comment (ivy-lli) [inline packages/editor/src/components/widgets/code-editor/CodeEditor.css:33]: is the font-size different for full size monaco?

Comment (ivy-edp) [reply packages/editor/src/components/widgets/code-editor/CodeEditor.css:33]: Well not really. It is also 12px but for some reason the Placeholder-Text is way to big if I don't set the font-size.

with no font-size:
grafik
with font-size:
grafik

Comment (ivy-lli) [reply packages/editor/src/components/widgets/code-editor/CodeEditor.css:33]: Then I think you can set it to the default placeholder css class right? would make more sense to me.

**Pull request closed without merging**